dht.go

```diff
@@ -119,14 +120,20 @@ func New(ctx context.Context, h host.Host, options ...opts.Option) (*IpfsDHT, er
 		// remove ourselves from network notifs.
 		dht.host.Network().StopNotify((*subscriberNotifee)(dht))
 
-		if dht.subscriptions.evtPeerIdentification != nil {
-			_ = dht.subscriptions.evtPeerIdentification.Close()
+		for _, sub := range []event.Subscription{
```

Comment:
lol, creative way to avoid an extra `!= nil` check without writing less code :P

Reply:
I could remove the line breaks between slice elements to reduce line count :P